scripts/zones/Jugner_Forest/mobs/Knight_Crab.lua

```diff
@@ -2,12 +2,21 @@
 -- Area: Jugner Forest
 --  MOB: Knight Crab
 -----------------------------------
+require("scripts/zones/Jugner_Forest/MobIDs");
 
 -----------------------------------
 -- onMobSpawn Action
 -----------------------------------
 
 function onMobSpawn(mob)
+
+    --If respawn and variable is not 0, then it respawned before someone killed all 10 crabs
```

Comment:
we should prolly just not allow that unless we know for sure retail does that?

Reply:
Yeah, I'll leave it since I remember it that way plain as day and the sites I read thus far mention similar functionality.